Purchases/Public/RCPurchases.m

```diff
@@ -108,6 +108,14 @@ + (void)setProxyURL:(nullable NSURL *)proxyURL {
     RCSystemInfo.proxyURL = proxyURL;
 }
 
++ (BOOL)forceUniversalAppStore {
+    return RCSystemInfo.forceUniversalAppStore;
+}
+
++ (void)setForceUniversalAppStore:(BOOL)forceUniversalAppStore {
+    RCSystemInfo.forceUniversalAppStore = forceUniversalAppStore;
+}
+
```

Comment:
couldn't come up with a way to write tests for this, since the default values live as macros on a header file, so there's no clean way of overriding / mocking them. Maybe this is just my brain being fried on a Friday afternoon. Open to ideas.

Reply:
I guess the only way would be running the test in different simulators? I don't think we can do it right now yes.